src/draw.cpp

```diff
@@ -123,7 +123,7 @@ void KDraw::set_window(SDL_Window* _window)
 void KDraw::blit2screen()
 {
     static int frame_count = 0;
-    static char fbuf[16] = "---";
+    static std::string fbuf = "---";
```

Comment:
This one isn't right because we `sprintf` to it on line 137.
Maybe use `std::to_string` ?